plugins/@grouparoo/iterable/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseTypes,
+} from "@grouparoo/core";
+import { connect } from "./../connect";
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod = async ({
+  appOptions,
+}) => {
+  const client = await connect(appOptions);
+  const required = getRequiredFields();
+  const known = await getUserFields(client);
+
+  return {
+    labels: {
+      property: {
+        singular: "Iterable Contact Property",
+        plural: "Iterable Contact Properties",
+      },
+      group: {
+        singular: "Iterable List",
+        plural: "Iterable Lists",
+      },
+    },
+    properties: {
+      required: required,
+      known: known,
+      allowOptionalFromProperties: true,
+    },
+  };
+};
+
+export const getRequiredFields = (): Array<{
+  key: string;
+  type: DestinationMappingOptionsResponseTypes;
+}> => {
+  return [{ key: "email", type: "string" }];
+};
+
+const mapTypesFromIterableToGrouparoo = (iterableType) => {
+  const map = {
```

Comment:
if there is a special "known" ones that should be a `phoneNumber`, can can pass the key in here and catch it earlier.

Reply:
It's usually a case statement at the top then the falls through to this mapping in the default type. You'd pass in the field name. SO if it's `phoneNumber` jsut said it's that type.